spec/the_google/calendar_test.rb

```diff
@@ -201,6 +201,21 @@
                 .returns api_result
         end
 
+        describe 'and the events do not have start or end values' do
+
+          let(:items) do
+            [
+              Struct.new(:summary, :start, :end).new(random_string, nil, nil),
+              Struct.new(:summary, :start, :end).new(random_string, nil, nil),
```

Comment:
In these cases, I think it's better to write the tests for the nil start and the nil end.  Isolate the test scenarios so you know both are covered.